package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "todo-list",
+  "version": "1.0.0",
+  "description": "A todo list app",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "webpack",
+    "watch": "webpack --watch"
```

Comment:
## Optional Change: Automation ♻️ 
- [ ] Do you mind adding an additional script for starting your project? Since we are using Webpack, we might as well utilise its power to run the most valuable scripts instead of doing some tasks manually. 
```suggestion
    "start": "webpack-dev-server --mode development --open" // optional`
```